lizmap/modules/lizmap/lib/Commands/WMTSCapabilities.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Lizmap\Commands;
+
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class WMTSCapabilities extends \Jelix\Scripts\ModuleCommandAbstract
+{
+    protected function configure()
+    {
+        $this
+            ->setName('wmts:capabilities')
+            ->setDescription('Get seeding capabilities')
+            ->setHelp('verbose option will provide tiles count for each TileMatrix')
+            ->addArgument('repository', InputArgument::REQUIRED, 'The repository ID')
+            ->addArgument('project', InputArgument::REQUIRED, 'The project name')
+            ->addArgument('layer', InputArgument::OPTIONAL, 'The layer name for which you want to know capabilities')
+            ->addArgument('TileMatrixSet', InputArgument::OPTIONAL, 'the TileMatrixSet (CRS) for which you want to know capabilites')
+
+        ;
+        // ->addOption('verbose', 'v',InputOption::VALUE_NONE, 'provide tiles count for each TileMatrix')
```

Comment:
```suggestion
            ->addOption('verbose', 'v', InputOption::VALUE_NONE, 'provide tiles count for each TileMatrix')
        ;
```
Define verbose option ?